Assets/Scripts/Enemy.cs

```diff
@@ -1,27 +1,49 @@
-﻿
-using System.Collections;
+﻿using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 
 public class Enemy : MonoBehaviour
 {
-    public int health = 10;
+    public int health;
+    public float speed;
+    public float flashTime;
+    public GameObject deadEnemyLand;
+    public GameObject particles;
+    Color originalColor;
+    public SpriteRenderer sRenderer;
+
+    void Start()
+    {
+        originalColor = sRenderer.color;
+    }
+
+    void Update()
+    {
+        if(health <= 0)
+        {
+            Destroy(gameObject);
+            Instantiate(deadEnemyLand, transform.position,Quaternion.identity);
+        }
+        transform.Translate(Vector2.left * speed * Time.deltaTime);
+    }
+
 
-    public GameObject deathEffect;
 
     public void TakeDamage(int damage)
```

Comment:
Could turn damage into a serialized variable so that you can tweak it easier.

Reply:
damage is currently a public variable that can be tweaked